README.md

```diff
@@ -15,25 +15,44 @@ For now, two constructors are exposed by the TU module:
 Documentation
 =============
 
+
+```js
+// Call the TableFilter constructor like so:
+
 TableFilter(tableElt, tolerance);
 
-TableFilter.prototype.addEvent();
-TableFilter.prototype.getTolerance();
-TableFilter.prototype.setTolerance(tolerance);
-TableFilter.prototype.filter(columnIndex, str);
+// Available functions
+
+TableFilter.addEvent(type, fn);
```

Comment:
This lets the reader think they are supposed to call functions on the constructor rather than on the created instance.

I think we should either let the `.prototype` in the function descriptions or call the methods on the return value of the `TableFilter(tableElt, tolerance);` call.

Like, either what you did before, or:

```js
var filter = TableFilter(tableElt, tolerance); // At the beginning of the README

filter.addEvent(type, fn); // And then show every function called on the created instance

Reply:
You have a point. I'll edit that in.